test/uk/gov/hmrc/customs/emailfrontend/services/DateTimeServiceSpec.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.services
+
+import org.joda.time.{DateTime, DateTimeZone}
+import java.time.{Clock, Instant, ZoneId, ZonedDateTime}
+import uk.gov.hmrc.customs.emailfrontend.services.DateTimeService
+import uk.gov.hmrc.customs.emailfrontend.utils.SpecBase
+
+
+class DateTimeServiceSpec extends SpecBase {
+
+  "DateTimeService" should {
+
+    "return current time in UTC using Joda Time" in {
+      val fixedInstant = Instant.parse("2023-10-31T12:00:00Z")
```

Comment:
some of this can probably be extracted out into a trait Setup as we're using it in both tests